emptytheme/block-templates/singular.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:template-part {"slug":"header","theme":"emptytheme","tagName":"header"} /-->
```

Comment:
The template parts are not loading on the edit-site screen but it is probably a bug with FSE.

Reply:
I think that's likely due to [the Gutenberg change noted here](https://github.com/WordPress/gutenberg/issues/27072#issuecomment-733108038). I loaded this up on a fresh site just now and it loads as expected.